nbextensions/bulkImportCell/tabs/multiAppInfo.js

```diff
@@ -44,11 +44,16 @@ define([
             for (const [fileType, status] of Object.entries(readyState)) {
                 fileTypeCompleted[fileType] = status === 'complete';
             }
-
+            const viewOnlyFileType = !['editingComplete', 'editingIncomplete'].includes(
```

Comment:
The other widget that uses the file type panel - it needs to know when to set that to viewOnly.